specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/logicalNetworks.json

```diff
@@ -0,0 +1,655 @@
+          "type": "string",
+          "readOnly": true,
+          "description": "Number of IP addresses allocated from the IP Pool"
+        },
+        "available": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Number of IP addresses available in the IP Pool"
+        }
+      }
+    },
+    "IPPool": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the IP-Pool"
+        },
+        "ipPoolType": {
+          "type": "string",
+          "enum": [
+            "vm",
+            "vippool"
+          ],
+          "x-ms-enum": {
+            "name": "IPPoolTypeEnum",
+            "modelAsString": false
+          },
+          "description": "Type of the IP Pool [vm, vippool]"
+        },
+        "start": {
+          "type": "string",
+          "description": "Start of the IP address pool"
+        },
+        "end": {
+          "type": "string",
+          "description": "End of the IP address pool"
+        },
+        "info": {
+          "$ref": "#/definitions/IPPoolInfo"
+        }
+      }
+    },
+    "RoutePropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "The destination CIDR to which the route applies."
+        },
+        "nextHopIpAddress": {
+          "type": "string",
+          "description": "The IP address packets should be forwarded to. Next hop values are only allowed in routes where the next hop type is VirtualAppliance."
+        }
+      },
+      "description": "RoutePropertiesFormat - Route resource."
+    },
+    "Route": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RoutePropertiesFormat",
+          "description": "Properties of the route."
+        },
+        "name": {
+          "type": "string",
+          "pattern": "^[a-zA-Z0-9]$|^[a-zA-Z0-9][-._a-zA-Z0-9]{0,78}[_a-zA-Z0-9]$",
+          "description": "Name - name of the subnet"
+        }
+      },
+      "description": "Route - Route resource."
+    },
+    "RouteTablePropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "routes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Route"
+          },
+          "description": "Collection of routes contained within a route table.",
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "RouteTablePropertiesFormat - Route Table resource."
+    },
+    "RouteTable": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Etag - Gets a unique read-only string that changes whenever the resource is updated."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name - READ-ONLY; Resource name."
```

Comment:
>Resource [](http://example.com/codeflow?start=44&length=8)

did you mean to add the           "readOnly": true, for this ?

Reply:
fixed